src/__tests__/skipwhile.test.ts

```diff
@@ -0,0 +1,9 @@
+import { of } from "../of";
+import { skipWhile } from "../skipwhile";
+import { toArray } from "../toArray";
+
+test("skipWhile", async () => {
+  const result = await toArray(
+    skipWhile((value: number) => value <= 2 || value >= 10 )(of(0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11)));
+  expect(result).toEqual([3, 4, 5, 6, 7, 8, 9]);
```

Comment:
this should be:

expect(result).toEqual([3, 4, 5, 6, 7, 8, 9, 10, 11])

because the skipWhile will stop as soon as the condition becomes true which happens when `value` is equals to 3 and it passes `value <= 2`

Reply:
you mean in skipwhile once we get the true result we don't have to check predicate further ?